### DISCUSSION TITLE
Pagination

### DISCUSSION THREAD

**Discussion by bhuppe** [derhansen/sf_event_mgt#1303] [Ideas]
Could be interesting to add pagination in search action, adding only the same variable:
'pagination' in modifySearchViewVariablesEvent, as in list action.
And <f:if condition="{pagination}"> in template as well
Thanks for this great extension!

**Comment by derhansen** [comment]
This is not as easy as it sounds, because the search demand is always user defined in frontend (e.g. search term, start- and/or enddate, ...) and all variable parameters must be part of the pagination parameters.

I always recommend to either use ext:solr or ext:ke_search for advanced requirements regarding search for events managed by the extension
